homeassistant/components/mobile_app/device_action.py

```diff
@@ -33,7 +253,60 @@ async def async_get_actions(
     if webhook_id is None or not supports_push(hass, webhook_id):
         return []
 
-    return [{CONF_DEVICE_ID: device_id, CONF_DOMAIN: DOMAIN, CONF_TYPE: "notify"}]
+    actions = [{CONF_DEVICE_ID: device_id, CONF_DOMAIN: DOMAIN, CONF_TYPE: "notify"}]
+
+    config = hass.data[DOMAIN][DATA_CONFIG_ENTRIES][webhook_id]
+    os_name = config.data[ATTR_OS_NAME]
+
+    if os_name == "Android":
+        actions += [
+            {
+                CONF_DEVICE_ID: device_id,
+                CONF_DOMAIN: DOMAIN,
+                CONF_TYPE: action,
+            }
+            for action in (
+                "command_stop_tts",
+                "command_update_sensors",
+                "request_location_update",
+                "command_auto_screen_brightness",
+                "command_bluetooth",
+                "command_ble_transmitter",
+                "command_beacon_monitor",
+                "command_activity",
+                "clear_notification",
+                "command_app_lock",
+                "command_broadcast_intent",
+                "command_dnd",
+                "command_high_accuracy_mode",
+                "command_launch_app",
+                "command_media",
+                "command_ringer_mode",
+                "command_screen_brightness_level",
+                "command_screen_off_timeout",
+                "command_screen_on",
+                "command_persistent_connection",
+                "command_volume_level",
+                "command_webview",
+                "remove_channel",
+            )
+        ]
+    elif os_name == "iOS":
```

Comment:
Can someone confirm what the OS name is for iOS devices?

I don't have an iOS device to test on, so have made a guess. I couldn't work it out for myself from this line, as it's using a system call:
https://github.com/home-assistant/iOS/blob/master/Sources/Shared/API/HAAPI.swift#L462